data/tlg4174/tlg001/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg4174" xml:lang="grc" urn="urn:cts:greekLit:tlg4174.tlg001">
+  <ti:title xml:lang="lat">Paraphrases in Dioynsium Periegetam</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg4174.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg4174.tlg001">
+    <ti:label xml:lang="lat">Paraphrases in Dioynsium Periegetam</ti:label>
+    <ti:description xml:lang="mul">Anonymous; Geographi graeci minores Volumen Secundum, ed. Karl Müller; Ambroise Firmin Didot, Paris, 1861</ti:description>
```

Comment:
Since I pull the metadata only from the description element for the Github Page, the title of the work should be there as well, i.e., "Paraphrases in Dioynsium Periegetam".